tools/build/helm.bzl

```diff
@@ -0,0 +1,17 @@
+load("@com_github_airyhq_bazel_tools//helm:helm.bzl", "helm_push_")
```

Comment:
Forgot to mention this, but you can also rename imports like so:
```
load("@com_github_airyhq_bazel_tools//helm:helm.bzl", lib_helm_push = "helm_push")
```

For when you want to avoid name collisions :)

Reply:
Ah cool, I will rename the function in the `bazel-tools` repo - only `helm_push` is more appropriate for it. :+1: